svg/README.md

```diff
@@ -59,7 +59,7 @@ export class AppComponent {
   ) {
     this.matIconRegistry.addSvgIcon(
       "breaker",
-      this.domSanitizer.bypassSecurityTrustResourceUrl(icon)
+      this.domSanitizer.bypassSecurityTrustResourceUrl(icon.default)
```

Comment:
Do we want to put the `.default` here, or would it be better in the import?

```
const iconSet = require(......).default;
```

Reply:
No preference on my end. Might be a little more clear if its on the import though. I'll change it up